pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go

```diff
@@ -108,6 +110,68 @@ type KeycloakAPIRealm struct {
 	// Require SSL
 	// +optional
 	SslRequired string `json:"sslRequired,omitempty"`
+
+	// Roles
+	// +optional
+	Roles *RolesRepresentation `json:"roles,omitempty"`
```

Comment:
*A generic comment, unrelated to this specific line.*

@slaskawi We currently don't support updating realms through the operator. Correct? This means users won't be able to manage realm roles after the realm was created, even though roles are not part of the "closest" realm setting in KC. Are we happy with this? Shouldn't we rather have realm roles as an independent CRD to better reflect KC's behaviour?

Reply:
@slaskawi Could you please comment on this?

Reply:
@vmuzikar The short answer is no. We do a one way sync. See more at https://groups.google.com/g/keycloak-dev/c/6OTpj3ow0u4/m/x2_qCAEEDAAJ